pkg/flagset/login.go

```diff
@@ -0,0 +1,54 @@
+package flagset
+
+import (
+	"github.com/micro/cli"
+	"github.com/owncloud/ocis/pkg/config"
+)
+
+// LoginWithConfig applies cfg to the root flagset
+func LoginWithConfig(cfg *config.Config) []cli.Flag {
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:        "client-id",
+			Value:       "cli",
+			Usage:       "oidc cli client id",
```

Comment:
needs descriptive text

Reply:
propose one

Reply:
Set the OpenID Client ID to send to the issuer